.travis.yml

```
@@ -0,0 +1,48 @@
+sudo: false
+dist: trusty
```

```suggestion
dist: bionic
```
Trusty is Ubuntu 14.04 --- sure it's an LTS release but there are three newer LTS's available: xenial bionic, and focal. I don't know what the status of Focal is on Travis yet but Bionic at least should work. Have you tried this against something newer? If it won't work that's a problem of it's own this project needs to work on and not a requirement for this PR, but we should at least _try_ the newest one possible.